assets/src/scripts/contributors.js

```diff
@@ -77,9 +77,9 @@ jQuery( function ( $ ) {
 			minWidth: xImg > xInit ? xInit : xImg,
 			minHeight: yImg > yInit ? yInit : yImg,
 			x1: x1,
```

Comment:
Here we can use `maxHeight` and `maxWidth` if we want to limit the cropped image resolution to a specific number of pixel. Ask to Steel.